linux_os/guide/services/ssh/ssh_server/sshd_disable_compression/bash/sle12.sh

```diff
@@ -0,0 +1,6 @@
+# platform = SUSE Linux Enterprise 12
+
+# Include source function library.
+. /usr/share/scap-security-guide/remediation_functions
+
+replace_or_append '/etc/ssh/sshd_config' '^Compression' 'delayed' '@CCENUM@' '%s %s'
```

Comment:
It would be better to have a variable `var_sshd_disable_compression` for this rather than a separate remediation script.